profiles/tests/test_audit_log.py

```diff
@@ -50,7 +50,9 @@ def assert_common_fields(
     assert audit_event["operation"] == operation
     assert audit_event["actor"]["role"] == actor_role
 
-    assert_almost_equal(audit_event["date_time_epoch"], now)
+    # 2000ms limit was arrived to by checking the maximum
+    # time at local laptop (~1000ms)
+    assert_almost_equal(audit_event["date_time_epoch"], now * 1000, 2000)
```

Comment:
These tests shouldn't use the `get_unix_timestamp_now` function anymore, because it returns only  a second precision timestamp. Instead something more accurate reference time should be used as `now`.

Reply:
> These tests shouldn't use the `get_unix_timestamp_now` function anymore, because it returns only a second precision timestamp. Instead something more accurate reference time should be used as `now`.

Good point. Such coarse testing would have hidden missing ms-component entirely. How about this, just switching to datetime?

Reply:
Yep.